GitUI/Strings.cs

```diff
@@ -22,6 +22,7 @@ internal sealed class Strings : Translate
         private readonly TranslationString _containedInNoBranchText = new TranslationString("Contained in no branch");
         private readonly TranslationString _containedInTagsText = new TranslationString("Contained in tags:");
         private readonly TranslationString _containedInNoTagText = new TranslationString("Contained in no tag");
+        private readonly TranslationString _invisibleCommitText = new TranslationString("'{0}' is not currently visible");
```

Comment:
```suggestion
        private readonly TranslationString _invisibleCommitText = new TranslationString("'{0}' is currently not visible");
```

Reply:
I'm not a native speaker, but to me this sounds odd. This kind of explains it: https://ell.stackexchange.com/a/228812